.cookiecutter/cookiecutter.json

```diff
@@ -8,6 +8,7 @@
     "report/__main__.py",
     "report/app.py",
     "report/core.py",
+    "tests/factories/*",
```

Comment:
I didn't think the cookiecutter's factories stuff (the utility function for hooking up `factory_boy` to an empty `sqlalchemy` session) was relevant to `report`, so just ignore it here and then the cookiecutter won't make any changes to `report`.